org.eclipse.jdt.ui/ui/org/eclipse/jdt/internal/ui/text/correction/ConvertFieldNamingConventionProposal.java

```diff
@@ -0,0 +1,145 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2012 IBM Corporation and others.
```

Comment:
Change to be: Copyright (c) 2023 IBM Corporation and others.

This is a new file so should have the current year only.